index.html

```diff
@@ -39,27 +40,38 @@
   <section id="introduction">
     <h2>Introduction</h2>
     <p>
-      See <a href="https://github.com/w3c/respec/wiki/User's-Guide">ReSpec's
-      user guide</a> for how to get started!
+      <em>This section is non-normative.</em>
+    </p>
+
+    <p>
+      Media is used extensively today, and the Web is one of the primary
+      means of consuming media content. Many browsers show content
+      recommendations in their UI. This specification aims to enable web
+      sites to specify a feed of personalized media recommendations that
+      can be fetched by the browser and displayed in the browser UI. By
+      improving media discovery this would improve the user experience
+      as well as providing web sites with a new way of reaching users.
     </p>
   </section>
-  <section data-dfn-for="Foo">
-    <h2>Start your spec!</h2>
-    <pre class="idl">
-    interface Foo {
-      attribute Bar bar;
-      void doTheFoo();
-    };
-    </pre>
+
+  <section id="conformance">
     <p>
-      See <a href="https://github.com/w3c/respec/wiki/WebIDL-Guide">ReSpec's
-      WebIDL guide</a> for how to write [[!WebIDL-2]] interfaces!
+      Requirements phrased in the imperative as part of algorithms (such as
+      "strip any leading space characters" or "return false and terminate
+      these steps") are to be interpreted with the meaning of the key word
+      ("must", "should", "may", etc) used in introducing the algorithm.
```

Comment:
This and the paragraph below, where are they coming from?

Reply:
It came from media session but I think we can remove it because respec should generate this for us